google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -164,8 +164,9 @@
                      key="unique.rest.signature.name" groupKey="inspections.group.name" enabledByDefault="true"
                      level="ERROR" implementationClass="com.google.cloud.tools.intellij.appengine.validation.RestSignatureInspection"/>
 
-    <postStartupActivity implementation="com.google.cloud.tools.intellij.PluginCompatibilityCheck"/>
-    <postStartupActivity implementation="com.google.cloud.tools.intellij.ConflictingAppEnginePluginCheck"/>
+    <postStartupActivity implementation="com.google.cloud.tools.intellij.startup.PluginCompatibilityCheck"/>
```

Comment:
We do currently check for the presence of the AE facet (see [below](https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/966/files#diff-98e94be8c6d76533d7afdc7ad8291fb1R71)) in order to determine whether the module is  an app engine module. We're need to access the `appengine-web.xml` in order to inform our logic about whether or not this is an appengine standard project (see `AppEngineProjectService.getInstance().getModuleAppEngineEnvironment(appengineWebXml)`). I'm not sure if there's a better way to do that.